controllers/clusters/cassandra_controller_test.go

```diff
@@ -37,86 +36,80 @@ const newCassandraNodeSize = "CAS-DEV-t4g.small-30"
 
 var _ = Describe("Cassandra Controller", func() {
 	var (
-		cassandraResource v1beta1.Cassandra
-		cassandraYAML     v1beta1.Cassandra
-		c                 = "cassandra"
-		ns                = "default"
-		cassandraNS       = types.NamespacedName{Name: c, Namespace: ns}
-		timeout           = time.Second * 40
-		interval          = time.Second * 2
+		ns = "default"
+
+		cassandra         v1beta1.Cassandra
+		cassandraManifest v1beta1.Cassandra
+
+		timeout  = time.Second * 40
+		interval = time.Second * 2
 	)
 
 	yfile, err := os.ReadFile("datatest/cassandra_v1beta1.yaml")
 	Expect(err).NotTo(HaveOccurred())
 
-	err = yaml.Unmarshal(yfile, &cassandraYAML)
+	err = yaml.Unmarshal(yfile, &cassandraManifest)
 	Expect(err).NotTo(HaveOccurred())
 
-	cassandraObjMeta := metav1.ObjectMeta{
-		Name:      c,
-		Namespace: ns,
-		Annotations: map[string]string{
-			models.ResourceStateAnnotation: models.CreatingEvent,
-		},
-	}
-
-	cassandraYAML.ObjectMeta = cassandraObjMeta
-
 	ctx := context.Background()
 
+	clusterID := cassandraManifest.Spec.Name + openapi.CreatedID
+	cassandraNS := types.NamespacedName{Name: cassandraManifest.ObjectMeta.Name, Namespace: ns}
```

Comment:
Please rename it because it is namespace and name, not just namespace

Reply:
NS just seems short and nice. I mean in terms of tests it's acceptable :sweat_smile: 
How about a variable name like `cassandraNNS`? :pineapple:

Reply:
Just to avoid further confusion, `ns` has been changed to `-NamespacedName`.

Reply:
Just to avoid further confusion, `-NS` has been changed to `-NamespacedName`.